libs/client/cornflow_client/constants.py

```diff
@@ -90,6 +90,74 @@
     pl.LpStatusUndefined: STATUS_UNDEFINED,
 }
 
+PARAMETER_SOLVER_TRANSLATING_MAPPING = {
+    ('bar_iter_limit', 'pyomo', 'cbc') : 'barr' ,
+    ('best_obj_stop', 'pyomo', 'cbc') : 'primalT' ,
+    ('cutoff', 'pyomo', 'cbc') : 'cuts' ,
+    ('iteration_limit', 'pyomo', 'cbc') : 'maxIt' ,
+    ('solution_limit', 'pyomo', 'cbc') : 'maxSaved' ,
+    ('time_limit', 'pyomo', 'cbc') : 'sec' ,
+    ('mip_gap', 'pyomo', 'cbc') : 'allow' ,
+    ('optimality_tol', 'pyomo', 'cbc') : 'dualT' ,
+    ('pump_passes', 'pyomo', 'cbc') : 'pumpC' ,
+    ('heuristics', 'pyomo', 'cbc') : 'heur' ,
+
+    ('is_mip', 'pulp', 'cbc') : 'mip' ,
+    ('optimality_tol', 'pulp', 'cbc') : 'gapAbs' ,
+    ('cutoff', 'pulp', 'cbc') : 'cuts' ,
+    ('time_limit', 'pulp', 'cbc') : 'timeLimit' ,
+    ('threads', 'pulp', 'cbc') : 'threads' ,
+    ('presolve', 'pulp', 'cbc') : 'presolve' ,
```

Comment:
How do you set the gap in pyomo? I do not see the option. Are you sure that you have implemented all the needed parameters?

Reply:
This is still unresolved. With cbc and pyomo I do not see the `gapAbs` parameter

Reply:
Because gapAbs parameter is defined from a SuperClass of CBC_solver in pulp called LP_solver. That means, gapAbs is a name that cbc decided to give to such a parameter but *does not exists* in cbc per se.
There are several option that can be discussed to set the optimality_tol. Among them, I chose dualT.